sources/packages/web/src/views/student/StudentAppealRequest.vue

```diff
@@ -3,11 +3,11 @@
     <template #header>
       <header-navigator title="Student" subTitle="Request a Change" />
```

Comment:
Minor. Having back button to student applications would be good even if temporary to avoid confusion. 
![image](https://github.com/user-attachments/assets/7321012b-5d32-4525-a990-66eaa011cc87)